pandas/tests/indexes/timedeltas/test_indexing.py

```diff
@@ -184,7 +184,8 @@ def test_take_fill_value(self):
         with pytest.raises(ValueError, match=msg):
             idx.take(np.array([1, 0, -5]), fill_value=True)
 
-        with pytest.raises(IndexError):
+        msg = "index -5 is out of bounds for size 3"
```

Comment:
looks like this message isnt quite right?

Reply:
Yes and no. There are OS differences. For Mac and I think Linux the message is `index -5 is out of bounds for **axis 0 with** size 3`. I've made the tests platform agnostic to look for that extra bit.